.github/workflows/integration_tests.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
 
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.9"
+          python-version: "3.10"
```

Comment:
Should we do this change in a different PR?

Reply:
makes sense